docs/manual/replacements_and_deletions.md

```diff
@@ -42,11 +42,45 @@ The following conditions are currently eligible for replacement:
 * `MissingPod`: This indicates a process group that doesn't have a Pod assigned.
 * `MissingPVC`: This indicates that a process group that doesn't have a PVC assigned.
 * `MissingService`: This indicates that a process group that doesn't have a Service assigned.
-* `PodPending`: This indicates that a process group where the pod is in a pending state.
+* `PodPending`: This indicates that a process group where the Pod is in a pending state.
+* `NodeTaintReplacing`: This indicates that a process group where the pod is running on a tainted Node
 
 Process groups that are set into the crash loop state with the `Buggify` setting won't be replaced by the operator.
 If the `cluster.Spec.Buggify.EmptyMonitorConf` setting is active the operator won't replace any process groups.
 
+## Automatic Replacements for Pods on Tainted Nodes
+The operator has an option to automatically replace Pods that are running on tainted Nodes. This feature is disabled by default but can be enabled by setting `automationOptions.replacements.taintReplacementOptions` and `automationOptions.replacements.taintReplacementTimeSeconds`.
+
+**Example 1.** We can use the following setup to let operator detect Pods running on Nodes with taint key `example.com/maintenance`, set the Pods' condition to `NodeTaintReplacing` if their Nodes have been tainted for 3600 seconds, and replace the Pods after 1800 seconds.
```

Comment:
```suggestion
### Example 1

The following setup lets the operator detect Pods running on Nodes with taint key `example.com/maintenance`, set the ProcessGroup' condition to `NodeTaintReplacing`, if their Nodes have been tainted for 3600 seconds, and replace the Pods after 1800 seconds.
```

Can we use proper markdown syntax and maybe a better/more precise heading than `example 1`? I personally find it a little bit hard to read this sentence.